examples/non-kubernetes/aws-otel-ec2-deployment.yaml

```diff
@@ -69,19 +69,21 @@ Resources:
                       send_batch_size: 50
                     batch/metrics:
                       timeout: 60s
+                    resourcedetection:
+                      detectors: [env, ec2]
+                      timeout: 5s
+                      override: true
                   exporters:
                     otlphttp:
-                      endpoint: ${sumo_http_traces_url}
+                      traces_endpoint: ${sumo_http_traces_url}
+                      tls:
+                        insecure: true
```

Comment:
Why do we need it? It should work with our public endpoint without setting `insecure: true`